.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: false
+language: python
+
+cache: pip
+
+# Tox cannot install pythons, so we set 3.5 as default, use 3.4 already
+# available in container and then include 3.6 in matrix.
+python:
+  - 3.5
+
+env:
+  - TOXENV=py34
+  - TOXENV=py35
+
+matrix:
+  include:
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.4  # i.e. skip installing 3.5
+      env: TOXENV=lint
+    - python: 3.4
+      env: TOXENV=coverage
+
+install:
+  # Temprorarily install dependencies until setup.py is fixed, see #383, #384:
+  - pip install pyaes rsa PyCrypto
+  - pip install tox
+
+script:
+  - tox
+
+after_success:
+  - if [[ $TOXENV =~ coverage ]]; then
+    pip install --quiet python-coveralls;
+    coveralls --ignore-errors;
```

Comment:
I hope this is not running the tests, as most won't pass.

Reply:
This one just submits coverage to coveralls. Tox does run the tests.